structs/forks/bellatrix/bellatrix.go

```diff
@@ -13,6 +13,7 @@ import (
 
 // BuilderSubmitBlockRequest spec: https://flashbots.notion.site/Relay-API-Spec-5fb0819366954962bc02e81cb33840f5#fa719683d4ae4a57bc3bf60e138b0dc6
 type SubmitBlockRequest struct {
+	BellatrixRaw              []byte           `json:"-"`
```

Comment:
have we checked it's not a problem in SSZ state root calculations? (asking for a friend ;> )

Reply:
How do we check this? In the testnet and Goerli the payloads are being delivered. That should be enough evidence